api/libproject/devstart.rb

```diff
@@ -1348,3 +1348,25 @@ def setup_cloud_project(cmd_name, *args)
   :description => "Initializes resources within a cloud project that has already been created",
   :fn => lambda { |*args| setup_cloud_project("setup-cloud-project", *args) }
 })
+
+def start_api_and_incremental_build(cmd_name, args)
```

Comment:
Do we want to do something similar for public-api? (Maybe in a follow-up?)

Reply:
Absolutely. If this approach seems to work for people, I'll happily add it there as well.